fh-messaging/lib/fh_rollup.js

```diff
@@ -12,6 +12,7 @@ exports.process = function(config, logger, inputDate, daysToKeep, cb) {
   logger.info(logMessage);
 
   var msgProcess;
+  
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the useless trailing whitespaces at the end of this line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ATrailingWhitespace)